pensjon-brevbaker-api-model/src/main/kotlin/no/nav/pensjon/brev/api/model/CommonEnums.kt

```diff
@@ -42,4 +42,100 @@ enum class Beregningsmetode{
     SOR_KOREA,
     SVEITS,
     USA
+}
+
+enum class KravAarsak {
```

Comment:
Har vi virkelig behov for å forholde oss til så mange forskjellige årsaker i malen?

Reply:
Jeg tror vi bør holde det 1:1 med pesys. Om det kommer nye kravårsaker som har noe med endringer å gjøre så må vi reagere på det i dette vedlegget. Jeg vet ikke helt om vi kan bryte det ned i enklere koder som reflekterer bruken i brevene enda. Mapper foreløpbig det med en when() statement i pesys.

Reply:
Foreløpig så ser det ut for meg som om det kun brukes til å finne ut en ting, så det kunne vært en boolean.

Reply:
Enig. Får ta det opp igjen når vi får flere bruk i en kontekst.

Reply:
Endret til boolean